config/template.config

```diff
@@ -8,15 +8,16 @@ includeConfig "${projectDir}/nextflow.config"
 
 // Inputs/parameters of the pipeline
 params {
-    algorithms = ['stats', 'collectwgsmetrics'] // 'stats', 'collectwgsmetrics', 'bamqc'
+    algorithm = ['fastqc', 'stats', 'collectwgsmetrics'] // 'fastqc', 'stats', 'collectwgsmetrics', 'bamqc'
     reference = '/hot/ref/reference/GRCh38-BI-20160721/Homo_sapiens_assembly38.fasta'
     output_dir = '/path/to/output/directory'
     blcds_registered_dataset = false // if you want the output to be registered
     save_intermediate_files = true
 
     // SAMtools stats options
-    samtools_remove_duplicates = false
-    samtools_stats_additional_options = ''
+    stats_max_rgs_per_sample = 20
```

Comment:
**suggestion**: Do we also want to add the max libs parameter here?

Reply:
@yashpatel6 I left it off in the interest of streamlining the template since library numbers that are large enough to be an issue are currently unlikely.   Do you want me to add it?

Reply:
For consistency, I would suggest adding it